.github/workflows/test.yml

```
@@ -32,4 +32,10 @@ jobs:
       - name: "check exported env vars"
         run: ./.github/workflows/scripts/test/test-envs.sh
       - name: "check action output JSON"
-        run: ./.github/workflows/scripts/test/test-json.sh '${{ steps.custom_versions_file.outputs.tools }}'
+        run: ./.github/workflows/scripts/test/test-json.sh '${{ steps.custom_versions_file.outputs.tools }}' false
+      - uses: ./
+        id: custom_versions_file_working_dir
+        with:
+          working_directory: __tests__/user-defined
+      - name: "check action output JSON - working_directory"
+        run: ./.github/workflows/scripts/test/test-json.sh '${{ steps.custom_versions_file_working_dir.outputs.tools }}' true
```

This test is going to fail because the bash script expect hard-coded versions. To avoid having to maintain 2 files in sync I would suggest to just use `__tests__` as the `working_directory` here. The end results should be the same.


```suggestion
        id: custom_versions_file_working_dir
        with:
          working_directory: __tests__
      - name: "check action output JSON - working_directory"
        run: ./.github/workflows/scripts/test/test-json.sh '${{ steps.custom_versions_file_working_dir.outputs.tools }}' true
```

That means also removing `__tests__/user-defined`